utils/prometheus-node-exporter-lua/Makefile

```diff
@@ -4,10 +4,10 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=prometheus-node-exporter-lua
-PKG_VERSION:=2017.12.08
```

Comment:
As this breaks compatibility I suggest to update the version

Reply:
agreed